docs/xailient_sdk_api.md

```diff
@@ -0,0 +1,86 @@
+# Xailient SDK API Documentation
+
+**Xailient SDK Version: 1.0.7**
+
+## dnn.Detector Class
+
+``` python
+detector = dnn.Detector(int)
+```
+
+#### Arguments
+
+| Arguments     | Input Type    | Default Value | Description   |
+| ------------- |:-------------:| :------------:| -------------|
+| cores _(optional)_         | int           | 1             |   Number of threads to start for running the Detector. This gives the ability to run multiple threads for inferencing. This value should be tuned and select based on the use case to maximise overall performance. |
+
+#### Example
+
+1. Using 2 threads
+
+    For example, your device has 4 cores and each core runs 1 process thread. If you want 1 thread to be running dedicatedly for camera inputs, one thread for post processing, then you can reserve two threads for the Detector so that it will perform efficiently.
+
+    This is how you would initialize the Detector so that it uses two cores:
+
+    ``` python
+    detector = dnn.Detector(2)
+    ```
+
+2. Using 1 thread
+
+    ``` python
+    detector = dnn.Detector()
+    ```
+
+    or 
+
+    ``` python
+    detector = dnn.Detector(1)
+    ```
+
+3. Using 4 threads
+
+    ``` python
+    detector = dnn.Detector(4)
+    ```
+
+### process_frame Method
+
+``` python
+_, bboxes = detector.process_frame(numpy.ndarray, int)
+```
+
+#### Arguments
+
+| Arguments     | Input Type    | Default Value | Description   |
+| ------------- |:-------------:| :------------:| -------------|
+| image         | numpy.ndarray           | -             |   Image as numpy array |
+| threshold _(optional)_         | int           | 0.4             |   Value between 0 and 1 for confidence score |
```

Comment:
int --> float

Reply:
Also, the default value for this threshold is now built into the SDK. So the default is not any fixed number. Should we say something like, "The default threshold is built into the SDK and is an optimal value determined when the model was trained."?